raiden-ts/tests/unit/fixtures.ts

```diff
@@ -125,3 +145,282 @@ export function epicFixtures(depsMock: MockRaidenEpicDeps) {
     state$,
   };
 }
+
+// fixture constants
+export const token = makeAddress();
+export const tokenNetwork = makeAddress();
```

Comment:
Would it make sense to use a `TOKEN_NETWORK` naming for the constacts?

Reply:
OK I saw below that you use them directly to avoid specifying the property name.